distro.go

```diff
@@ -171,11 +188,21 @@ func (d Distro) GetConfiguration() (c Configuration, e error) {
 
 // String deserializes a distro and its configuration as a yaml string.
 func (d Distro) String() string {
+	return fmt.Sprintf("name: %s\n%s\n%s", d.Name(), d.guidToString(), d.configToString())
+}
+
+func (d Distro) guidToString() string {
+	id, err := d.GUID()
+	if err != nil {
+		return fmt.Sprintf("guid: |\n  %v", err)
+	}
+	return fmt.Sprintf("guid: '%v'", id)
+}
+
+func (d Distro) configToString() string {
```

Comment:
Same remark on "They are probably the same thing" for that method.

Reply:
So, let’s turn that to "add a comment"

Reply:
Done